stories/Technical Planning Documents/Audit of Storybook Component Documentation.mdx

```diff
@@ -0,0 +1,230 @@
+import '../styles.css';
+
+# Audit of Storybook Component Documentation
+
+#### By Michael Larocca - github: [@MichaelLarocca](https://github.com/MichaelLarocca)
+
+---
+
+## About
+
+#### **This documentation provides an overview of the audit process to identify components lacking comprehensive Storybook stories. The goal is to ensure all components are thoroughly documented and tested within Storybook, enhancing our codebase's overall quality and maintainability.**
+
+### Components
+
+<table>
+  <thead>
+    <tr>
+      <th>Component Name</th>
+      <th>Sufficient Story</th>
+      <th>Insufficient Story</th>
```

Comment:
What are we using to consider a story insufficient?

Reply:
There are components with previous Stories written that must meet Shashi's standards. 

The 'Insufficient Storyfield' allows us to identify current and future Stories that must be rewritten to meet the required standards. 

(Per Shashi, we will be following the carbon design system for GIS Story standards)